examples/Spirit/spirit_utils.cc

```diff
@@ -510,6 +509,55 @@ double calcWork(
   return work;
 }
 
+
+template <typename T>
+double calcElectricalWork(
+    drake::multibody::MultibodyPlant<T> & plant,
+    std::vector<drake::trajectories::PiecewisePolynomial<double>>& x_trajs,
+    drake::trajectories::PiecewisePolynomial<double>& u_traj,
+    double efficiency){
+
+  auto velocities_map = multibody::makeNameToVelocitiesMap(plant);
+  auto actuator_map = multibody::makeNameToActuatorsMap(plant);
+  int n_q = plant.num_positions();
+
+  double work = 0;
+  double Q = 0;
+  for(const auto& x_traj : x_trajs) {
+    std::vector<double> knot_points = x_traj.get_segment_times();
+    for (int knot_index = 0; knot_index < knot_points.size() - 1; knot_index++) {
+      auto u_low = u_traj.value(knot_points[knot_index]);
+      auto u_up = u_traj.value(knot_points[knot_index + 1]);
+      auto x_low = x_traj.value(knot_points[knot_index]);
+      auto x_up = x_traj.value(knot_points[knot_index + 1]);
+
+      for (int joint = 0; joint < 12; joint++) {
+
+        if(joint == 1 or joint == 3 or joint == 5 or joint == 7)
+          Q = 0.249;
```

Comment:
I would add a comment making it clear which versoin of spirit this comes from and maybe an equation. Or actually calculate these values with all the motor properties. This Q with no description will likely bite us or someone else in the but later.

Reply:
Maybe we should include motor porperties and gear ratios in the utils file? or at least remember it as a TODO for a spirit class in the future

Reply:
I added a constant to the utils file for Q